### PR TITLE
4th stable Commit

### PR DESCRIPTION
**Pull request by cq-mohtashim-ali**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,27 +1,17 @@
 const fs = require("fs");
 const WebSocket = require("ws").Server;
 const mime = require("mime");
-const { decodeBase64 } = require("bcryptjs");
```
Comment (MasoodAhmad) [inline src/app.js:4]: why deleted?

File: src/components/checkUser.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function checkUser(name) {
+const UserArray = [{ name: "Ali" }, { name: "Ahmad" }, { name: "Asad" }];
```
Comment (MasoodAhmad) [inline src/components/checkUser.js:2]: still names

File: src/app.js [hunk 3]
```diff
@@ -30,26 +20,16 @@ wss.on("connection", (ws) => {
...
-    const fileName = `${data.name}-${year}-${month}-${day}-${hours}${minutes}-${seconds}-${timeStamp}.png`;
-    const p = path.join(__dirname, `../images/${data.name}/${year}/${month}/${day}/${fileName}`);
-    fs.mkdirSync(ImgaesDir, { recursive: true });
-    fs.writeFileSync(p, images, "utf8");
+   
 
+    const timedNames = getTimedNames.getTimedName(data.name);
+    const p = path.join(__dirname, timedNames[1]);
+    fs.mkdirSync(timedNames[0], { recursive: true });
+    fs.writeFileSync(p, images, "utf8");
```
Comment (MasoodAhmad) [inline src/app.js:29]: this was meant to be async

File: src/app.js (hunk 1 above)
Comment (cq-mohtashim-ali) [reply src/app.js:4]: Wasn't using that at all.

File: src/app.js (hunk 3 above)
Comment (cq-mohtashim-ali) [reply src/app.js:29]: Ok

File: src/components/checkUser.js (hunk 2 above)
Comment (cq-mohtashim-ali) [reply src/components/checkUser.js:2]: Ok changing that

**Pull request merged**